docs/site/themes/template/layouts/partials/resources-case-1.html

```
@@ -1,7 +1,7 @@
 {{ $data := sort .Site.Data.resources ".use_case_order" }}
 {{ range $index, $element := $data }}
     {{ if eq .use_case_number 1 }}
-    <li>
+    <li class="d-none">
```

hidden by default to prevent the sections from contracting (showing then hiding) when the page loads